examples/light-switch-app/silabs/include/LightSwitchMgr.h

```diff
@@ -45,6 +48,8 @@ class LightSwitchMgr
     void GenericSwitchOnShortRelease();
 
     void TriggerLightSwitchAction(LightSwitchAction action, bool isGroupCommand = false);
+    void TriggerLevelControlAction(uint8_t level, bool isGroupCommand = false);
+    uint8_t currentLevel;
```

Comment:
Why do we need to store a copy of the current level instead of using the attribute that is already persisted?

Reply:
I didn't add attributes to the Level Control cluster, should i include the current_level attribute of cluster and use that for storing level?

Reply:
Again like state in other comments, it doesn't make any sense for the light-switch to store a current level this way.